ui/src/stores/utils/contract.js

```diff
@@ -53,8 +53,8 @@ export const getBalanceOf = async (contract, address) => {
   return fromDecimals(balance, decimals)
 }
 
-export const mintedTotally = async contract => {
```

Comment:
is it possible to reflect this in tests?

Reply:
We could add some unit tests for this method, mocking the returned value from `contract.methods.mintedTotallyByBridge(bridgeAddress).call()` and also unit tests for `HomeStore` that could make sure that `mintedTotallyByBridge` method is called from the store

Reply:
yes, do this please

Reply:
Done

Reply:
thanks